src/cpp/core/json/JsonRpc.cpp

```diff
@@ -324,16 +323,15 @@ void setJsonRpcResponse(const JsonRpcResponse& jsonRpcResponse,
    {
       LOG_ERROR(error);
       pResponse->setError(http::status::InternalServerError,
-                          error.code().message());
+                          error.getMessage());
    }
 }     
 
 bool JsonRpcResponse::parse(const std::string& input,
                             JsonRpcResponse* pResponse)
 {
    Value value;
-   bool valid = json::parse(input, &value);
-   if (!valid)
+   if (!!value.parse(input))
```

Comment:
Double negative?

Reply:
Yes, it parse returns an Error instead of a boolean value now, so no error is good and error is bad (hence the swap), and the double negative just ensures that we're invoking the boolean operator on the Error class, which is marked explicit.

Reply:
IIRC testing Error directly in this fashion was still safe through the safe bool idiom:

https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Safe_bool

Is that not the case?

Reply:
Or is this more a restriction imposed by the launcher SDK requirements?

Reply:
Oh no, I think I can remove the double negatives. Now to find them all.